From 39e66117e444c2c98059e98774b2fe7e12480d15 Mon Sep 17 00:00:00 2001
From: Roberto Oliveira <robertoguimaraes8@gmail.com>
Date: Mon, 2 Apr 2018 10:55:37 +0000
Subject: lxd: add missing limits.h include

On systems without glibc, as Alpine, you might lack definition of PATH_MAX.
This patch adds the limits.h header to solve this issue.

Signed-off-by: Roberto Oliveira <robertoguimaraes8@gmail.com>
---
 lxd/main_forkfile.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lxd/main_forkfile.go b/lxd/main_forkfile.go
index 73f470ca..6c68c896 100644
--- a/lxd/main_forkfile.go
+++ b/lxd/main_forkfile.go
@@ -17,6 +17,7 @@ import (
 #include <string.h>
 #include <sys/stat.h>
 #include <unistd.h>
+#include <limits.h>
 
 extern char* advance_arg(bool required);
 extern void error(char *msg);
